dask_kubernetes/operator/customresources/daskautoscaler.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: autoscaler.kubernetes.dask.org
+spec:
+  scope: Namespaced
+  group: kubernetes.dask.org
+  names:
+    kind: Autoscaler
+    plural: autoscalers
+    singular: autoscaler
+    shortNames:
+      - as
+  versions:
+    - name: v1
+      served: true
+      storage: true
+      schema:
```

Comment:
Does this need a reference to the worker group that it will be scaling?

Reply:
Yes, I would make sure it works with the default worker group, then add that reference later.